rlp/src/rlpin.rs

```diff
@@ -189,6 +189,15 @@ impl<'a> Rlp<'a> {
 	pub fn at<'view>(&'view self, index: usize) -> Result<Rlp<'a>, DecoderError>
 	where
 		'a: 'view,
+	{
+		let (rlp, _offset) = self.at_with_offset(index)?;
+		Ok(rlp)
+	}
+
+	pub fn at_with_offset<'view>(&'view self, index: usize)
```

Comment:
What is the purpose of adding this as a new method rather than changing the return value of `at()`? Just to reduce the breakage in `parity-ethereum`? If yes then I think I'd prefer to do a breaking release of `rlp` and do the grunt work.

Reply:
Well, most of the time you don't need the offset, so even if designing the API from scratch I think having two separate methods probably makes sense. But yeah, probably the main reason is backwards compatibility. Just doesn't seem like there's a good reason to break the current API, IMO.

Reply:
> most of the time you don't need the offset,

In your `eth` branch it's used 4 times and only for `RlpNodeCodec` so I guess you're right. `at()` is used much more extensively, which kind of surprised me: I'd have hoped it was used much less.